android/src/main/java/com/internxt/mobilesdk/services/crypto/Hash.kt

```diff
@@ -0,0 +1,41 @@
+package com.internxt.mobilesdk.services.crypto
+
+import org.spongycastle.crypto.digests.RIPEMD160Digest
+import org.spongycastle.jcajce.provider.digest.RIPEMD160
+import java.security.MessageDigest
+
+class Hash {
+
+  /**
+   * Given an array of bytes, generates a SHA256 Hash in bytes
+   */
+  public fun sha256(input: ByteArray): ByteArray {
+    val digest = MessageDigest.getInstance("SHA-256")
```

Comment:
`hasher` could be a better naming

Reply:
you mean for the file name? or the function name? sha256Hasher?

Reply:
for the val name:

`val hasher`

As is the hasher (who hashes) who digests the content